--- a/CMakeLists.txt	2026-07-07 17:08:36.568126336 -0500
+++ a/CMakeLists.txt	2026-07-07 17:09:41.940128664 -0500
@@ -76,7 +76,7 @@
         RESULT_VARIABLE RESULT
         OUTPUT_VARIABLE GIT_COMMIT_HASH)
 
-    string (STRIP ${GIT_BRANCH} GIT_BRANCH)
+    string (STRIP "${GIT_BRANCH}" GIT_BRANCH)
     string (STRIP ${GIT_COMMIT_HASH} GIT_COMMIT_HASH)
 
     message(STATUS "vym: git branch: ${OUTPUT} - commit: ${GIT_COMMIT_HASH}")
